roles/convert/tasks/verify_conversion.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Default conversion_inhibited to false
+  ansible.builtin.set_fact:
+    conversion_inhibited: false
+
+- name: Collect JSON report results
+  ansible.builtin.slurp:
+    src: "{{ parse_c2r_report_result_filename_json }}"
+  register: results_json
+
+- name: Parse report results
+  ansible.builtin.set_fact:
+    convert2rhel_report_json: "{{ results_json.content | b64decode | from_json }}"
+
+- name: Check for SKIP, OVERRIDABLE, ERROR statuses
+  ansible.builtin.set_fact:
+    conversion_inhibited: true
+  when:
+    # All of these are unnaceptable based on https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/preconversion_assessment_script.py#L121
+    - "'SKIP' == item.result.level or 'SKIP' in item.msg_levels"
+    - "'OVERRIDABLE' == item.result.level or 'OVERRIDABLE' in item.msg_levels"
+    - "'ERROR' == item.result.level or 'ERROR' in item.msg_levels"
+
+  loop: "{{ report_actions }}"
+  vars:
+    report_actions: "{{ convert2rhel_report_json.actions | map('combine', {'msg_levels': item.messages | map(attribute='level')}) | convert2rhel_report_json.actions }}"
```

Comment:
overall it's fine by me, maybe to have the task only once, it could be extracted to separate file in parse role and then imported via relative path... take it more like suggestions (to have the code only in one place).. some articles even sugggested separate role, but that feel like too much for this use case.

I would be fine even with copying it, it gets the job done. Will leave that to one more reviewer.

Reply:
I think that's good idea, I will do just that. Thanks @andywaltlova. I don't like repeating code also. ;)

Reply:
after looking at it more, the conversion_inhibited should be available even in convert role so you were right I don't need to copy the code. Just checking the fact that the conversion was inhibited.

Reply:
:thinking: hmm I kinda like the approach but I don't get how it should work? I think we can have two approches:

1. Have analysis role (also includes parse role) which sets the inhibited variable as a dependency of convert role = it will run before and the variable will be set and available - this would need to be changed in `meta/main.yml` to have analysis included in dependencies 
2. Have the conversion role undependent on analysis, and just run it, if it fails with exit code 1 then somehow check for the inhibitors - but minor headache here could be caused by the fact that technically convert2rhel can fail with exit code 1 with whatever fail it doesn't automatically mean that it was inhibited, the report tight to that given run has to be checked.

Maybe the first approach would be easier, WDYT @r0x0d ? 
Another plus could be that with the first approach we would know that if the conversion (after analysis) fails with exit code 1 it isn't inhibited and something went wrong.

Or am I missing something? It is entirely possible that I just don't see the connection how should this be enough to work properly.

Reply:
well afaik there is common, analysis and convert role, analysis now import common role (covert role should probably import common as well), but the analysis is a task used in analysis role and that tasks imports the parse role